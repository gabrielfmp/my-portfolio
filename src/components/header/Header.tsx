import { Description, Name, StyledHeader } from './styles';
import React from 'react';

const Header = () => (
  <StyledHeader>
    <Description>
      Hi! I’m a frontend engineer driven by inovation and meaningful
      connections. I focus mainly on improving every user’s experience with
      simple yet effective code,
      <s>even though most of the time I’m solving bugs I created myself.</s>
    </Description>

    <Name>
      Gabriel
      <br />
      Pires
    </Name>
  </StyledHeader>
);

export default Header;
