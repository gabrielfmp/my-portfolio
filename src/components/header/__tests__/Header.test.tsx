import { render } from '@testing-library/react';
import Header from '../Header';
import React from 'react';

test('snapshot', () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});
