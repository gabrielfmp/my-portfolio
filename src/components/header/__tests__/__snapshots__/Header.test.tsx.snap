// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <header
    class="css-1wskg7y"
  >
    <p
      class="css-1ooz59o"
    >
      Hi! I’m a frontend engineer driven by inovation and meaningful connections. I focus mainly on improving every user’s experience with simple yet effective code,
      <s>
        even though most of the time I’m solving bugs I created myself.
      </s>
    </p>
    <h1
      class="css-13fdacc"
    >
      Gabriel
      <br />
      Pires
    </h1>
  </header>
</DocumentFragment>
`;
